servicebox-app/src/main/resources/templates/ClaimPage/list.html

```diff
@@ -36,14 +36,14 @@ <h1>Claims</h1>
                 <tbody id="claim-list">
                   {#for claim in claims}
                       <!-- Include the HTML template rendering the claim item -->
-                      {#include ClaimExtendedResource/item.html claim=claim /}
+                      {#include ClaimPage/item.html claim=claim /}
```

Comment:
Rename according to the resource renaming

Reply:
OK. Let's do that